app/src/main/java/at/bitfire/icsdroid/ui/AddCalendarDetailsFragment.kt

```diff
@@ -66,7 +64,42 @@ class AddCalendarDetailsFragment: Fragment() {
             Toast.makeText(requireActivity(), message, Toast.LENGTH_LONG).show()
         }
 
-        return v
+        val colorPickerContract = registerForActivityResult(ColorPickerActivity.Contract()) { color ->
```

Comment:
Shouldn't this be moved to the top level of the class? Instead of inside the `onCreateView` function?

Reply:
Not sure. We don't use it anywhere else at the moment, so I see no need to put it far away from where it is used. Why would you do want to do that?

Reply:
Allthough we are going to get rid of the fragment entirely at some point as well no ... ? :wink:

Reply:
Yup, true, I'll merge now 😉